src/utility/index.jsx

```diff
@@ -124,30 +118,67 @@ export const getOrdinal = (n) => {
 
 const getSubtext = (row) => {
   if (row.match_id && row.player_slot !== undefined) {
-    // TODO localize
-    return isRadiant(row.player_slot) ? 'Radiant' : 'Dire';
+    return isRadiant(row.player_slot) ? strings.radiant : strings.dire;
   }
   if (row.last_played) {
     return <FromNowTooltip timestamp={row.last_played} />;
   }
+  if ((row.name && row.value) !== undefined) {
```

Comment:
why not just check start_time?  If you don't want to render any subtext, that should be determined by the parent component.